content/spin/v2/javascript-components.md

```diff
@@ -18,13 +17,14 @@ url = "https://github.com/fermyon/developer/blob/main/content/spin/v2/javascript
 - [Storing Data in SQLite](#storing-data-in-sqlite)
 - [Storing Data in MySQL and PostgreSQL Relational Databases](#storing-data-in-mysql-and-postgresql-relational-databases)
 - [AI Inferencing From JS/TS Components](#ai-inferencing-from-jsts-components)
+- [`Node.js` Compatibility](#nodejs-compatibility)
 - [Using External NPM Libraries](#using-external-npm-libraries)
   - [Suggested Libraries for Common Tasks](#suggested-libraries-for-common-tasks)
 - [Caveats](#caveats)
 
-With JavaScript being a very popular language, Spin provides support for building components with it using the experimental SDK. The development of the JavaScript SDK is continually being worked on to improve user experience and add features. 
+With JavaScript being a very popular language, Spin provides an SDK to support building components. The development of the JavaScript SDK is continually being worked on to improve user experience and add features. The SDK is based on [ComponentizeJS](https://github.com/bytecodealliance/ComponentizeJS).
```

Comment:
In the previous section, it was `ComponentizeJS` in backticks. We should be consistent.